web/ugahacks-6/next.config.js

```diff
@@ -1,3 +0,0 @@
-module.exports = {
```

Comment:
Let's keep this config file and refactor the recap packet redirect logic as an async redirects function that returns the correct path -> destination since the redirect will be static and this keeps our approach consistent across the workspaces. See https://nextjs.org/docs/api-reference/next.config.js/redirects or the next.config.ts file in the ugahacks-7 workspace for reference.